Badge/index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="../general.css">
+    <link rel="stylesheet" href="../allstyle.css">
+    <link rel="stylesheet" href="style.css">
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.3.0/font/bootstrap-icons.css"
+    />
+    <title>Avatar</title>
+</head>
+<body>
+  <a href="#">News <span class="badge">SALE</span></a><br>
+  <a href="#">Comments <span class="badge">10</span></a><br>
+  <a href="#">Updates <span class="badge">2</span></a>
+
+  <button class="btn btn-primary">Primary <span class="badge">5</span></button>
+  
+
+<p>Below is a demonstration how you can use badge with Avatar transform property</p>
+<div>
+    <img src="https://robohash.org/pratyush.png?set=set2" alt="robot image" class="avatar"> <span class="badge" style ="transform: translateX(-30px);";">2</span>
+<div/>
+        <p>Below is a demonstration how you can use badge with icontransform property</p>
+    <i class="bi bi-bag-fill" style="font-size: 70px;"><span class="badge" style ="transform: translate(-20px,10px);";">2</span></i>
```

Comment:
i think you should remove inline style because it will not be available to the user of your CL.

Reply:
Thank you for the feedback. I'll make the changes.